pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.ruff]
 
-select = ["E", "F", "W", "C", "S"]
+select = ["E", "F", "W", "C", "Q", "SLF", "I"]
```

Comment:
What does this line do?

Reply:
It enables more linters. All of them can be listed by using `ruff linter`. Q - check string quotes, SLF - self. param check, I - import sorting

Reply:
Can you make the linter change a separate PR? I want Ben and others to discuss that.

Reply:
yes sure, I'll delete this commit ASAP

Reply:
done